WordPress/src/main/java/org/wordpress/android/ui/reader/discover/viewholders/ReaderPostNewViewHolder.kt

```diff
@@ -89,6 +89,7 @@ class ReaderPostNewViewHolder(
     }
 
     private fun updateInteractionCountsSection(state: ReaderPostNewUiState) = with(binding) {
+        // here
```

Comment:
I suppose we can delete this one? I also leave comments in key places while developing 😅